### DISCUSSION TITLE
Exception handling?

### DISCUSSION THREAD

**Discussion by baohx2000** [spatie/fork#15] [Q&A] [answered]
Really dig the nice fluent interface.
Just curious how exceptions or errors are/could be handled? Exit codes?

**Comment by brendt** [comment, marked as answer]
There's no special exception handling in place, that's outside the scope of this package. If an exception is thrown in a child process, it should be handled in there. If you want cross-process exception handling, you probably want to look at https://github.com/spatie/async
